### PR TITLE
Improve EHE info and buff lava in EHE

### PR DESCRIPTION
**Pull request by StaffiX**
1. Buff lava and make it produce sc and sh steam instead of sh steam and regular steam
8x more lava throughput (20k -> 160k)
image
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14517

2. Add structure blocks to structure tooltip list
image
as part of https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17847

3. Adds more description to explain how EHE works 
image

4. Removes threshold with plasma input and removes overheat water input since with the removal of steam from lava, all recipes uses the same amount of water regardless if its normal or overheated
image

### CODE REVIEW

**Comment by FireWarrior2** [comment]
Can string "Max Distilled Water Input(Overheated):" be also fixed not to get out from window?
For example, make something like:
Max Distilled Water:
Normal - 20,000 L/s
Overheated - 20,000 L/s
This will also probably make Threshold string not get on next recipe (4 strings - > 3 strings).

**Comment by StaffiX** [comment]
kép
well I tried but it looks confusing with the threshold, and this doesn't really help with lack of EHE's documentation.
My issue is that this way, it doesnt really look like that line 1, 2 and 5 are separate, while line 3, 4 and 5 are connected

**Comment by FireWarrior2** [comment]
Just move threshold after hot fluid input.
Maybe rename it to "Hot Fluid Threshold"

**Comment by StaffiX** [comment]
This ended up being more than just a simple bug fix

File: src/main/java/goodgenerator/loader/RecipeLoader2.java [hunk 1]
```diff
@@ -1064,12 +1064,12 @@ public static void RecipeLoad() {
...
-            FluidRegistry.getFluidStack("ic2distilledwater", 20000),
-            FluidRegistry.getFluidStack("steam", 3200000),
-            FluidRegistry.getFluidStack("ic2superheatedsteam", 1600000),
-            10000);
+            FluidRegistry.getFluidStack("lava", 160000),
+            FluidRegistry.getFluidStack("ic2pahoehoelava", 160000),
+            FluidRegistry.getFluidStack("ic2distilledwater", 80000),
+            FluidRegistry.getFluidStack("ic2superheatedsteam", 12800000),
+            FluidRegistry.getFluidStack("supercriticalsteam", 12800000),
+            80000);
```
Comment (serenibyss) [inline src/main/java/goodgenerator/loader/RecipeLoader2.java:1072]: Is this just a display recipe?

Comment (serenibyss) [reply src/main/java/goodgenerator/loader/RecipeLoader2.java:1072]: Discussed in discord

**Review by serenibyss — APPROVED** [review]

**Pull request merged**